e2e/call-done-twice/__tests__/index.test.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+describe('`done()` called more than once', () => {
+  it('should fail', done => {
+    done();
+    done();
+  });
+
+  it('should fail inside a promise', done => {
+    Promise.resolve()
+      .then(() => {
+        done();
+        done();
+      })
+      .catch(err => err);
+  });
+});
+
+describe('multiple `done()` inside a hook', () => {
+  beforeEach(done => {
```

Comment:
just for completeness, could you add `afterEach`, `beforeAll` and `afterAll` as well? separate `describe`s probably

Reply:
With `afterAll`, it doesn't seem to work. The test suite fails to run. 
![image](https://user-images.githubusercontent.com/40818234/101630568-112b3500-3a49-11eb-925b-b1d1f07e8a2c.png)

Reply:
yeah, errors in afterAll should do that. does that invalidate the other tests somehow?

Reply:
No, everything else is fine. I'll push the changes then!